### PR TITLE
Add calendar and time range availability sliders to main site

### PR DESCRIPTION
**Pull request by sfirrin**

### CODE REVIEW

**Comment by sasjkia** [comment]
Thanks @sfirrin - looks like this branch has some conflicts?

File: index.html [hunk 1]
```diff
@@ -230,7 +254,7 @@ <h3 class="panel-title">I need a ride</h3>
                                 </div>
 
                                 <div class="form-group">
-                                    <label for="rideLocations" class="col-sm-2 control-label">Check Your Polling Location</label>
+                                    <!--<label for="rideLocations" class="col-sm-2 control-label">Check Your Polling Location</label>-->
```
Comment (rickr) [inline index.html]: Is this comment needed?

**Comment by sasjkia** [comment]
This is really important... Partner has already emailed to push twice. She really needs a demo. By when do you think this could be incorporated?

**Comment by dmilet** [comment]
Looking at index.html trying to understand what it means for the back end...
1. Store interval values in those hidden inputs in ISO 8601 format, rather than some custom representation.
2. The class of the hidden input should be a constant value in order for the back end to correctly identify them
3. Use the Id attribute in the DOM to store timeId

**Comment by dmilet** [comment]
After further reading and testing...
1. the hidden input riderDateTimeRangesJSON is constructed by the web page, and therefore can be used as-is by the node.js app and the field stored in the database. However, I really wonder why we would store the data this way {"month":"Oct","day":"7","startTime":"10:10am","endTime":"5:50pm"} rather than using ISO 8601 format.

2. The form from the master_calendar branch has only on field for Name, while the database expects two fields, one for the First Name the other for the Last Name.

3. Node.js app expects driver forms to be submitted to <url>/driver and similarly rider forms to be submitted to <url>/rider. The index.html must be changed anyway to not use CloudStitch anymore.

4. We don't need to duplicate the information in hidden inputs named riderDateTimeRangesJSON and datetime[].

**Comment by misteranderson** [comment]
I'm working on this item - in order to get rider and driver datetimeranges into a reasonable format.

**Pull request closed without merging**